### ISSUE TITLE
Use of assume-no-moving-gc

### ISSUE THREAD

**Issue by nleiva** [karimra/gnmic#563]
FYI. I run into the following when running the API package with Go1.18.

```bash
Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.18 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.18 set. Notably, if go1.18 adds a moving garbage collector, this program is unsafe to use.
```

I didn't dig why you are using it, but it looks it might be in more than one place.

```bash
$ go mod why go4.org/unsafe/assume-no-moving-gc
# go4.org/unsafe/assume-no-moving-gc
gnmi
github.com/karimra/gnmic/api
github.com/karimra/gnmic/utils
github.com/hairyhenderson/gomplate/v3
github.com/hairyhenderson/gomplate/v3/funcs
inet.af/netaddr
go4.org/intern
go4.org/unsafe/assume-no-moving-gc
```

```bash
$ go list -m all | grep ^go4.org
go4.org/intern v0.0.0-20210108033219-3eb7198706b2
go4.org/unsafe/assume-no-moving-gc v0.0.0-20201222180813-1025295fd063
```

**Comment by karimra** [comment]
Thanks for pointing this out.
I haven't tried go1.18 yet but I will look into it.

The `go4.org/unsafe/assume-no-moving-gc` package seems be coming from `github.com/hairyhenderson/gomplate/v3/funcs`,
which I use to add custom functions to all Go templates used by gNMIc.
The [net functions](https://docs.gomplate.ca/functions/net/) in `github.com/hairyhenderson/gomplate` happen to use `inet.af/netaddr`

I will keep an eye on this, thanks again

**Comment by nleiva** [comment]
`inet.af/netaddr` is the precursor of `net/netip` that comes in Go1.18. I'll look into `github.com/hairyhenderson/gomplate` to understand what their plans are.

**Comment by hellt** [comment]
I think gomplate author has it all under control
https://github.com/hairyhenderson/gomplate/pull/1335

**Comment by karimra** [comment]
Hi, sorry this took some time to get to but using the latest tag `v0.25.0` you should not see this issue anymore.

**Comment by nleiva** [comment]
Thanks @karimra. I was on `v0.25.1` and upgrading to `v0.27.1` didn't help. 

Are you saying you made changes to the old `v0.25.0`? Any chance you can tag it as `v0.25.2` or else?

Thanks

**Comment by hellt** [comment]
These version numbers you posted @nleiva make me think that you were upgrading containerlab and not gnmic

Gnmic is at 0.25.1 version right now

**Comment by nleiva** [comment]
OMG, you are right. I’m still recovering from covid as you can tell. Sorry for the misunderstanding

**Comment by karimra** [comment]
No worries, were you able to make it work with 0.25.x ?

**Comment by nleiva** [comment]
Yep, now that I clicked the right button it seems to fix the problem. Thank you!
